projects/plugins/jetpack/extensions/blocks/ai-paragraph/deprecated/v1.js

```diff
@@ -0,0 +1,45 @@
+import { useBlockProps } from '@wordpress/block-editor';
+import { pasteHandler } from '@wordpress/blocks';
+
+/**
+ * Example data to migrate:
+ * ```
+ * <!-- wp:paragraph -->
+ * <p>GPT generates a block for testing.  This block should be an interesting but somewhat obscure fact regarding pirates and their constant battles with ninjas.</p>
```

Comment:
I could add some more comments for this.  But essentially if you open the code editor, & paste this in it will try to convert the blocks to the new version.

Reply:
Prettier doesn't like me making it easy to copy/paste so I'm just going to remove.